test/utility_clopper_pearson_interval_test.cpp

```diff
@@ -19,17 +19,17 @@ int main() {
 
   clopper_pearson_interval<> iv(deviation{1});
 
-  // {
-  //   const auto x = iv(0, 1);
-  //   BOOST_TEST_IS_CLOSE(x.first, 0.0, deps);
-  //   BOOST_TEST_IS_CLOSE(x.second, 0.8413447460685429, deps);
-  // }
-
-  // {
-  //   const auto x = iv(1, 0);
-  //   BOOST_TEST_IS_CLOSE(x.first, 0.1586552539314571, deps);
-  //   BOOST_TEST_IS_CLOSE(x.second, 1.0, deps);
-  // }
+  {
+    const auto x = iv(1, 1);
+    BOOST_TEST_IS_CLOSE(x.first, 0.0, deps);
+    BOOST_TEST_IS_CLOSE(x.second, 0.8413447460685429, deps);
+  }
+
+  {
+    const auto x = iv(1, 1);
+    BOOST_TEST_IS_CLOSE(x.first, 0.1586552539314571, deps);
+    BOOST_TEST_IS_CLOSE(x.second, 1.0, deps);
+  }
```

Comment:
Why this change? You should not change the original test condition, because that should have been correct.

Reply:
This is a mistake on my end, I changed this while finding where the issue came from (and realized it wasn't this) and missed changing it back. I see you've fixed it.